target_stitch/__init__.py

```diff
@@ -194,7 +271,12 @@ def persist_lines(stitchclient, lines):
             state = message.value
 
         elif isinstance(message, singer.SchemaMessage):
-            schemas[message.stream] = message.schema
+
+            # Draft4Validator will fail unless both the multipleOf in the
+            # schema and the value in the data are Decimal. So we need to
+            # convert all instances of multipleOf in the schema to
+            # Decimal.
+            schemas[message.stream] = ensure_multipleof_is_decimal(message.schema)
```

Comment:
It's to take advantage of functionality in the validator. Draft4Validator will respect the multipleOf keyword when doing validation, but the `multipleOf` and the value in the data have to have the same type. So for example `0.12345` would not validate against `{"multipleOf": 0.1}`. But if `0.12345` is a decimal and `0.1` is a float it will throw an exception.